arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -236,11 +248,43 @@
           "nextLinkName": null
         }
       }
+    },
+    "/providers/Microsoft.Sql/operations": {
```

Comment:
I don't think we have a way to reference the operation from one place, so if it lists all the operations and work, the generated SDK code will include the API on each client.